data/strings/strings.txt

```diff
@@ -13692,6 +13692,51 @@
     zh-Hans = 登录
     zh-Hant = 登入
 
+  [not_signed_in]
+    tags = android
+    en = Not signed in
+    af = Nie aangemeld nie
+    ar = لم يتم تسجيل الدخول
+    az = Daxil deyil
+    be = Не ўвайшлі
+    bg = Не е влязъл в системата
+    ca = No s'ha iniciat la sessió
+    cs = Nepřihlášen
+    da = Ikke logget ind
+    de = Nicht eingeloggt
+    el = Δεν έχετε συνδεθεί
+    es = No iniciado sesión
+    et = Ei ole sisse logitud
+    eu = Saioa hasi gabe
+    fa = وارد نشده است
+    fi = Ei kirjautunut sisään
+    fr = Non signé
```

Comment:
For French, it would be "Non connecté•e" fir gebder neutral ^^

Reply:
thanks 👍️